test/evergreen.yml

```diff
@@ -2326,7 +2298,7 @@ tasks:
             set -o verbose
             virtualenv -p python3 venv
             source venv/bin/activate
-            pip3 install gcovr
+            pip3 install lxml==4.8.0 Pygments==2.11.2 Jinja2==3.0.3 gcovr==5.0
```

Comment:
Can you share the details of the problem it ran into without pinning the dependency package version for gcovr?

Reply:
Yep, this was a similar issue to what we hit with WT-9044: `UnknownLineType: function __modify_apply_one called 1604355 returned NAN % blocks executed 80%`. I compared WT-9044's failing commit to its successful predecessor and the change in gcovr version was the cause.
There's a resolved issue on the gcovr repo, but taking the latest gcovr from pip doesn't fix this issue so I've downgraded to 5.0 which also required specifying `lxml`, `Pygments`, `Jinja2`.

As a general note I'd be in favour of pinning all of our pip dependencies to avoid errors like this, or at least catching them when we manually upgrade.

Reply:
Thanks for the detailed information! Mind adding a comment in Jira for easier search later on?